iOS Example Frame SPM/iOS Example Frame Integration Tests/CardTypeTokenCreationTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  iOS_Example_Frame_Integration_Tests.swift
+//  iOS Example Frame Integration Tests
+//
+//  Created by Okhan Okbay on 17/07/2023.
+//  Copyright © 2023 Checkout. All rights reserved.
+//
+
+import XCTest
+import Frames
+
+final class iOS_Example_Frame_Integration_Tests: XCTestCase {
+
+    func testTokenCreation() throws {
```

Comment:
Why is this func having throws?

Reply:
it's started with Xcode 11.4 and the default first method comes as a throwing function. As far as I'm aware, if it throws it shows as a test failure. But in this case, it's not needed and removed now